common/common-interactor/src/common_sovereign_interactor.rs

```diff
@@ -1470,11 +1471,16 @@ pub trait CommonInteractorTrait: InteractorHelpers {
             .register_sovereign_token(hash_of_hashes, token)
             .returns(ReturnsHandledOrError::new())
             .returns(ReturnsNewTokenIdentifier)
+            .returns(ReturnsLogs)
             .run()
             .await;
 
         self.assert_expected_error_message(response, None);
 
+        let expected_logs =
+            vec![log!(TRANSFER_VALUE_ONLY_LOG, topics: [""], data: Some(token_id.clone()))];
```

Comment:
I don't think this is ok because we don't transfer any tokens in the transaction.
We need to use the approach from [here](https://github.com/multiversx/mx-sovereign-sc/blob/fd0d435a4cb4e4a7150061c52595c71effef611f/mvx-esdt-safe/tests/mvx_esdt_safe_blackbox_setup.rs#L392) with error+expected_logs in args and stuff, but we need to use logs when the config.SHARD_1 or something like that